tests/common.rs

```diff
@@ -175,6 +186,28 @@ pub fn load_cairo_path(program_path: &str) -> (String, Program, SierraCasmRunner
     (module_name.to_string(), program, runner)
 }
 
+/// Compiles a cairo starknet contract from the given path
+pub fn load_cairo_contract_path(path: &str) -> ContractClass {
+    let mut db = RootDatabase::builder()
+        .detect_corelib()
+        .with_plugin_suite(starknet_plugin_suite())
+        .build()
+        .unwrap();
+
+    let main_crate_ids = setup_project(&mut db, Path::new(&path)).unwrap();
+
+    compile_contract_in_prepared_db(
+        &db,
+        None,
+        main_crate_ids.clone(),
+        CompilerConfig {
+            replace_ids: true,
+            ..Default::default()
+        },
+    )
+    .unwrap()
```

Comment:
In general I think these unwraps are ok since they are mostly part of the testing framework, but still it's still questionable not to use expect with a proper message instead. Within reason, please change the unwraps to expect with a message stating what failed or the underlying assumption that was violated when panicking.

Reply:
Done!